fml/ascii_trie.cc

```diff
@@ -0,0 +1,69 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#include "flutter/fml/ascii_trie.h"
+#include "flutter/fml/logging.h"
+
+namespace fml {
+typedef AsciiTrie::TrieNode TrieNode;
+
+namespace {
+void Add(TrieNode** trie, const char* entry) {
+  int ch = entry[0];
+  FML_DCHECK(ch < 128);
```

Comment:
Define a constant for the number of children per node.  (128 is hardcoded in other places, like `FreeNode`)

Or use a `std::vector` or `std::array` for `children`